packages/generator-superset/README.md

```diff
@@ -0,0 +1,32 @@
+# generator-superset
+
+[![Version](https://img.shields.io/npm/v/@superset-ui/generator-superset.svg?style=flat)](https://img.shields.io/npm/v/@superset-ui/generator-superset.svg?style=flat-square)
+[![David (path)](https://img.shields.io/david/apache-superset/superset-ui.svg?path=packages%2Fgenerator-superset&style=flat-square)](https://david-dm.org/apache-superset/superset-ui?path=packages/generator-superset)
+
+> Scaffolder for Superset
+
+## Installation
+
+First, install [Yeoman](http://yeoman.io) and `generator-superset` using [npm](https://www.npmjs.com/) (we assume you have pre-installed [node.js](https://nodejs.org/)).
+
+```bash
+npm install -g yo
+npm install -g @superset-ui/generator-superset
+```
+
+## Usage
+
+Generate a new package in `@superset-ui`
+
+```bash
+cd superset-ui/packages
+mkdir superset-ui-new-package
+cd superset-ui-new-package
+yo @superset-ui/superset
```

Comment:
is this line correct? not `@superset-ui/generator-superset`?

Reply:
Yes, it is correct. `yo xxx` looks for global npm packages (or `npm link`'ed package) named `generator-xxx`. 

You will notice all `yeoman` generator package names start with `generator-`

This is why I am tempted to name this package `generator-superset` without `@superset-ui` so we can call `yo superset`, but that will not published under the npm org.

Reply:
ah got it, thanks for the explanation 👍  

I think using the `@superset-ui` namespace would still be good.